client/components/domains/use-my-domain/domain-input.jsx

```diff
@@ -54,21 +54,18 @@ function UseMyDomainInput( {
 		? __( 'Enter the domain you would like to use:' )
 		: __( 'Enter your domain here' );
 
-	const hasDomainNoteLabel =
-		[ 'en', 'en-gb' ].includes( locale ) ||
-		hasTranslation(
-			'Please enter your site address without “www” or “https://” e.g. mydomain.com'
-		);
-	const domainNote = hasDomainNoteLabel
-		? __( 'Please enter your site address without “www” or “https://” e.g. mydomain.com' )
-		: '';
-
 	const hasDomainInputNoteLabel =
 		[ 'en', 'en-gb' ].includes( locale ) ||
 		hasTranslation( 'Enter the domain you would like to use:' );
 	const domainInputNote = hasDomainInputNoteLabel
 		? __( 'This won’t affect your existing site.' )
 		: '';
+
+	const hasDomainPlaceholderLabel =
+		[ 'en', 'en-gb' ].includes( locale ) || hasTranslation( 'yoursiteaddress.com' );
```

Comment:
See example domains to use PCYsg-eI0-p2 — yoursiteaddress.com is owned by some domain re-seller.

Reply:
Question; since we use specific example domains, do we typically translate them?

Reply:
Nice catch! Thank you!

I replaced it with `yourgroovydomain.com` from that document.

Reply:
> Question; since we use specific example domains, do we typically translate them?
I'm not sure, the previous one (mydomain.com) was translated. So I went along with it.

Reply:
If the website would be wrapped in a translate call, the translators might translate it.
For example - translations in GlotPress for [example.com](https://translate.wordpress.com/projects/wpcom/-all-translated/233557/) and [mydomain.com](https://translate.wordpress.com/projects/wpcom/-all-translated/853096/).

Translated website name would create a better localized experience, but we'd probably want to add a translators comment to instruct the translators to either leave it or translate to a standard 'example' site in their language, if it exists.